### PR TITLE
Postmerge fixing Ecology chapter

### PR DESCRIPTION
**Pull request by jannes-m**

### CODE REVIEW

File: 15-eco.Rmd [hunk 1]
```diff
@@ -519,7 +519,7 @@ search_space = paradox::ps(
 Having defined the search space, we are all set for specifying our tuning via the `AutoTuner()` function.
 Since we deal with geographic data, we will again make use of spatial cross-validation to tune the hyperparameters\index{hyperparameter} (see Sections \@ref(intro-cv) and \@ref(spatial-cv-with-mlr)).
 Specifically, we will use a five-fold spatial partitioning with only one repetition (`rsmp()`). 
-In each of these spatial partitions, we run 50 models (`trm()`) while using randomly selected hyperparameter configurations (`tnr`) within predefined limits (`seach_space`) to find the optimal hyperparameter\index{hyperparameter} combination.
+In each of these spatial partitions, we run 50 models (`trm()`) while using randomly selected hyperparameter configurations (`tnr()`) within predefined limits (`seach_space`) to find the optimal hyperparameter\index{hyperparameter} combination.
```
Comment (Robinlovelace) [inline 15-eco.Rmd:522]: Definite improvement, follow-on question, worth explaining in more detail what these functions are, I'm new to them and am not sure from this good but terse description.

File: 15-eco.Rmd [hunk 2]
```diff
@@ -562,12 +562,12 @@ saveRDS(at, "extdata/15-tune.rds")
 ```
 
 ```{r 15-eco-26, echo=FALSE, eval=FALSE}
-tune = readRDS("extdata/15-tune.rds")
+at = readRDS("extdata/15-tune.rds")
```
Comment (Robinlovelace) [inline 15-eco.Rmd:565]: What does `at` stand for? Autotune? may be worth stating that somewhere or using a longer and more descriptive object name.

**Review by Robinlovelace — APPROVED** [review]

**Comment by Robinlovelace** [comment]
Approved as some definite changes in there. Feel free to merge @jannes-m to keep the momentum going I think 'go fast and break things' is a reasonable attitude. The build seems to be failing with this message:

```
Error: [rast] cannot open file: /vsicurl/https://zenodo.org/record/5774954/files/clm_snow.prob_esacci.dec_p.90_500m_s0..0cm_2000..2012_v2.0.tif
```

Source: https://github.com/Robinlovelace/geocompr/runs/6077814566?check_suite_focus=true#step:4:5548

File: 15-eco.Rmd (hunk 1 above)
Comment (jannes-m) [reply 15-eco.Rmd:522]: Good point, will reference the spatial-cv chapter as I have explained there in a little more detail how to construct an `AutoTuner()`.

File: 15-eco.Rmd (hunk 2 above)
Comment (jannes-m) [reply 15-eco.Rmd:565]: yes, `at` stands for AutoTuner and I guess this was more obvious while creating the object: https://github.com/Robinlovelace/geocompr/blob/672991f23a7115b77c8ee199bdf8353b84f289f0/15-eco.Rmd#L526-L538
Still, the name could be more descriptive, something like `autotuner_rf` where rf stands for random forest.

Comment (Robinlovelace) [reply 15-eco.Rmd:565]: > Still, the name could be more descriptive, something like `autotuner_rf` where rf stands for random forest.

Agreed, I see now the tests are unrelated to your changes Jannes, so I suggest merging this now to keep the momentum. Many thanks!

**Comment by jannes-m** [comment]
give me a few minutes, then I will merge

**Comment by jannes-m** [comment]
And yes, the momentum is only there because the baby is overdue -:)

**Pull request merged**